solosis/commands/alignment/cellranger_vdj.py

```diff
@@ -0,0 +1,110 @@
+import logging
+import os
+import tempfile
+
+import click
+
+from solosis.utils.input_utils import collect_samples
+from solosis.utils.logging_utils import debug
+from solosis.utils.lsf_utils import lsf_options_std, submit_lsf_job_array
+from solosis.utils.state import logger
+
+FASTQ_EXTENSIONS = [".fastq", ".fastq.gz"]
+
+
+@lsf_options_std
+@debug
+@click.command("cellranger-vdj")
+@click.option("--sample", type=str, help="Sample ID (string)")
+@click.option(
+    "--samplefile",
+    type=click.Path(exists=True),
+    help="Path to a CSV or TSV file containing sample IDs",
+)
+# @click.option(
```

Comment:
If this option is no longer relevant (i.e. it doesn't exist in the `vdj` pipeline), then this should be removed.